app/models/main.py

```diff
@@ -60,6 +60,35 @@ def __init__(self, astext_type=None):
         super(JSON, self).__init__(none_as_null=True, astext_type=astext_type)
 
 
+class RemovePersonalDataModel:
```

Comment:
As discussed, let's rename this to make it clearer that it's a mixin and not its own table (and update the docstring for the method).

Reply:
Done 👍